Makefile

```diff
@@ -71,6 +71,26 @@ GO ?= CGO_ENABLED=0 go
 # Allow setting of go build flags from the command line.
 GOFLAGS := 
 
+# Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
```

Comment:
is controller-gen a go program? Can it also be found in the Go path bin directory?

Reply:
This is the same code as the Kubebuilder Makefile, but I think because it is related to our installation it should be in our local bin.